src/expression/evaluation_context.ts

```diff
@@ -29,8 +40,33 @@ class EvaluationContext {
         return this.feature && 'id' in this.feature ? this.feature.id : null;
     }
 
+    geometryDollarType() {
+        return this.feature ?
+            typeof this.feature.type === 'number' ? geometryTypes[this.feature.type] : simpleGeometryType[this.feature.type] :
+            null;
```

Comment:
According to coverage report, `null` is never returned. Is it possible that it is dead code? I can't think of a way to the code to reach it...

Reply:
The `null` was in the original expression.  I guess it is a safety net in case the method is called when the context object has no feature.

The coverage report is a result if a new line break. I'm OK with a longer line, if you prefer.

Reply:
I think that if something is wrong it will return a null before this null return value, so I think it's safe to remove it.

Reply:
Never mind, My mistake, null is for when the feature is null...